abb_irb120_support/CMakeLists.txt

```diff
@@ -12,8 +13,7 @@ if (CATKIN_ENABLE_TESTING)
     DEPENDENCIES
       # not nice, but industrial_robot_client doesn't prefix its targets
       joint_trajectory_action
-      abb_driver_robot_state
-      abb_driver_motion_download_interface)
+      abb_driver_robot_state)
```

Comment:
`abb_driver_motion_download_interface` is the name of the target in `abb/abb_driver/CMakeLists.txt` that builds the `motion_download_interface` binary.

But as I wrote, the whole `DEPENDENCIES` arg list should be removed as it doesn't work as intended.